Reflect latest SDK changes (0.13.5)

calintje opened the pull request:
- Updated examples to support Token 2022 Program based position NFTs
- Added new `closePosition` method to the close position script
- Included Token 2022 Program token accounts in 04* series
- Update tx to Versioned Transaction in 013* and 014*
- ~~Updated JP examples~~
- ~~Manually tested scripts 011 - 081, EN + JP~~
- Update `./generate_localization.sh` to transform relative path `../wallet.json` to `../../wallet.json` in generated files

calintje committed:
Update open position instructions to use TE based positions NFTs. Update close_position script. Add versioned transactions to 01* series.

calintje committed:
Update open position instructions to use TE based positions NFTs. Update close_position script. Add versioned transactions to 01* series.

calintje committed:
clean up

calintje committed:
Update files in src_i18n and autogenerate src/EN/* and src/JP/*. Update generate_localization.sh.

yugure-orca commented on generate_localization.sh:
```diff
@@ -30,6 +30,6 @@ do
     do
         echo "  generate: $TARGET_DIR/$lang/$f"
-        grep -v -E $filter $SRC_DIR/$f | sed $command > $TARGET_DIR/$lang/$f
+        grep -v -E $filter $SRC_DIR/$f | sed -e "$command" -e 's|import secret from "\.\./wallet.json"|import secret from "\.\./\.\./wallet.json"|g' > $TARGET_DIR/$lang/$f
```
nit: `sed -e "$remove_lang_prefix" -e "$replace_wallet_path"` may be reader friendly

yugure-orca commented on src_i18n/041_get_positions.ts:
```diff
@@ -23,12 +22,14 @@ async function main() {
 
   //LANG:JP 全てのトークンアカウントを取得
   //LANG:EN Get all token accounts
-  const token_accounts = (await ctx.connection.getTokenAccountsByOwner(ctx.wallet.publicKey, {programId: TOKEN_PROGRAM_ID})).value;
```
Could you add the following comment here for 041 and 042.

```
LANG:JP このチュートリアルや UI は Token-2022 を使った NFT を作ります。Token を使った古い NFT も念の為探索します。
LANG:EN This tutorial and UI create NFTs using Token-2022. We will also explore older NFTs created with the traditional Token standard, just in case.
```

I think it is nice that we explain why devs need to search NFTs with both Token-2022 and Token.

yugure-orca commented on src_i18n/081_close_position.ts:
```diff
@@ -40,189 +32,21 @@ async function main() {
-  //LANG:EN Wait for the transaction to complete
-  const latest_blockhash = await ctx.connection.getLatestBlockhash();
-  await ctx.connection.confirmTransaction({signature, ...latest_blockhash}, "confirmed");
+  for (const tx of close_position_tx) {
```
Could you add the following comment here ?
```
//LANG:JP Close は手数料やリワードの回収など必要な処理を多く含むため、2つ以上のトランザクションに分割される場合があります。
//LANG:JP ただし、多くの場合は1つのトランザクションであることを期待できます。
//LANG:EN Since Close includes various necessary processes such as fee and reward collection, it may be split into two or more transactions.
//LANG:EN However, in most cases, you can expect it to be a single transaction.
```

yugure-orca reviewed (approved):
@calintje 
Thanks for the update!

I added some nit comments, but LGTM!
Please feel free to merge the PR (after the tweak & re-generation EN/JP)

note: I've sent contributor invitation to you. 👍️

calintje committed:
Resolve comments

calintje commented:
Thank you so much for the review! The additional inline comments make it indeed more clear for users. I added the comments and updated the `generate_localization` script.
